### DISCUSSION TITLE
feature: ability to perform traffic splitting and canarying of services

### DISCUSSION THREAD

**Discussion by hbagdi** [Kong/kong#7328] [Ideas and feature requests]
### Summary

Open-source Kong should be able to perform traffic-splitting or A/B testing of services. This is also known as canarying of services.

I'm aware that this is an enterprise feature currently. The scope of this issue is to track the development of this feature in Open Source Kong.

**Comment by vaidik** [comment]
Hi

is there an update on by when can we expect this feature to go out?

**Comment by mayankshah1607** [comment]
Hi @hbagdi 

Any idea when this feature might be available on open source Kong?
Thanks :)

**Comment by ulyssescb** [comment]
Any news here? :)
